### DISCUSSION TITLE
[Request] Add Option for Mail Cow Lxc

### DISCUSSION THREAD

**Discussion by Dvalin21** [tteck/Proxmox#2979] [Ideas]
Requesting to Mail Cow as a proxmox lxc.

Docker version https://docs.mailcow.email/
and https://docs.mailcow.email/getstarted/install/#docker-and-docker-compose-installation

**Comment by Fabick75** [comment]
This is a very interesting project, but undokerizing this seems a big effort, because is based on a lot of containers, it's not only one program that need to be correctly installed

**Comment by protonaut** [comment]
I think this is helpfull for installing mailcow in an LXC on proxmox:
https://community.mailcow.email/d/1413-howto-install-mailcow-in-a-lxc-container-on-proxmox-64-13-for-home-usage

Looking forward to a script.

**Comment by PH1L900D** [comment]
Yes I second this request

**Comment by LittleDummy420** [comment]
+1 🍰  https://github.com/Mrhudson69/Mailcow-Setup-inside-LXC-Proxmox
